### ISSUE TITLE
2nd room level design change

### ISSUE THREAD

**Issue by cosme12** [egordorichev/platformer#4]
![pico-8_0](https://user-images.githubusercontent.com/6611118/29199040-cccc5b34-7e1f-11e7-88ab-6bbe25ddc142.png)

I think the first hole should be deeper, so the player gets forced to jump and learn how it works before the spikes. Also, the block over the column should be higher, so the player don't get smashed there.

**Comment by egordorichev** [comment]
That's true. Also, this room should teach the player about changing the worlds.
